### PR TITLE
add grid, refactor: toast component

### PR DESCRIPTION
**Pull request by iRohitGaur**

### CODE REVIEW

**Review by varun13169 — CHANGES_REQUESTED** [review]
On the Toast Page, GRID is not cliackable

**Review by varun13169 — APPROVED** [review]

File: pages/components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  grid-template-columns: repeat(11, 1fr);
+}
+.grid_h10 {
+  grid-template-columns: repeat(10, 1fr);
+}
+.grid_h9 {
+  grid-template-columns: repeat(9, 1fr);
+}
+.grid_h8 {
+  grid-template-columns: repeat(8, 1fr);
```
Comment (manojsarna) [inline pages/components/grid/grid.css:18]: good classification of classes

File: pages/components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+          <div>g=10</div>
+          <div>g=10</div>
+          <div>g=10</div>
+          <div>g=10</div>
+        </div>
+        <div class="sui_grid grid_h9">
+          <div>g=9</div>
+          <div>g=9</div>
+          <div>g=9</div>
+          <div>g=9</div>
```
Comment (manojsarna) [inline pages/components/grid/grid.html:105]: looks good and elegant

**Review by manojsarna — APPROVED** [review]
overall code is looking good. 

its clean.

good naming class conventions

**Review by sudipkundu999 — APPROVED** [review]
Looks so clean! Good to merge.
Taking a lot of inspiration from this!

**Pull request merged**